lib/widgets/investigation_details.dart

```diff
@@ -0,0 +1,127 @@
+import 'package:connect2bahmni/domain/models/omrs_order.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import '../utils/debouncer.dart';
+
+class InvestigationDetails extends StatefulWidget {
+  final OmrsOrder investigation;
+  const InvestigationDetails({super.key, required this.investigation});
+
+  @override
+  State<InvestigationDetails> createState() => _InvestigationDetailsState();
+}
+
+class _InvestigationDetailsState extends State<InvestigationDetails> {
+  OmrsOrder _omrsOrder = OmrsOrder();
+  final TextEditingController _notesController = TextEditingController();
+  final Debouncer _debouncer = Debouncer();
+  OmrsOrderType? _selectedOrderType;
+@override
+  void initState(){
+    super.initState();
+  }
+
+  @override
+  void dispose() {
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    var arg = ModalRoute.of(context)!.settings.arguments;
```

Comment:
Why is this required .. you are anyways calling InvestigationDetails with an instance of "OmrsOrder" .. so this is redundant.